src/components/ReportContact.jsx

```diff
@@ -1,83 +1,84 @@
 import React from 'react';
-import { withStyles } from '@material-ui/core/styles';
+import styled from 'styled-components';
 import { ArrowBack } from '@material-ui/icons';
-import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
 import PropTypes from 'prop-types';
 
+import { 
+  Typography,
+  Button,
+} from '@material-ui/core';
 
-const styles = {
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: 16px;
+`;
 
-  container: {
-    display: 'flex',
-    alignItems: 'center',
-    marginBottom: 16,
-  },
+const ButtonStyled = styled(Button)`
+  && {
+  min-width: 46px;
+  width: 46px;
+  height: 46px;
+  background-color: #46A440;
+  margin-right: 16px;
+  color: white;
+  }
+`;
 
-  button: {
-    minWidth: 46,
-    width: 46,
-    height: 46,
-    backgroundColor: '#46A440',
-    marginRight: 16,
-    color: 'white',
-  },
+const Text = styled(Typography)`
+  && {
+  color: white;
+  font-size: 1.7rem;
+  }
+`;
 
-  link: {
-    textDecoration: 'none',
-    color: 'white',
-  },
+const TextSmall = styled(Typography)`
+  && {
+  color: white;
+  font-size: 1.4rem;
+ }
+`;
 
-  text: {
-    color: 'white',
-    fontSize: '1.7rem',
-  },
-
-  textSmall: {
-    color: 'white',
-    fontSize: '1.4rem',
-  },
-
-  ref: {
-    color: 'white',
-    textDecoration: 'none',
-    '&:hover': {
-      color: '#CACACA',
-    },
-  },
-
-};
+const Refs = styled.a`
+  && {
+    color: white;
+    text-decoration: none;
+     &:hover 
```

Comment:
Check indentation here. Seems to be only one space.

Reply:
@schalkventer; Commit- 91d6a21 resolves.